pkg/apis/proto/sideinput/v1/sideinput.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+option go_package = "github.com/numaproj/numaflow-go/pkg/apis/proto/sideinput/v1";
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+package sideinput.v1;
+
+service UserDefinedSideInput {
```

Comment:
```suggestion
UserDefinedSideInput is used to broadcast changes to all the vertices ...... <fill it up>..
service UserDefinedSideInput {
```

Could you please document the expected behavior and use of this proto file? You may link to other documents.